### PR TITLE
feat: show log conversation conflict modal when editing or saving log

### PR DESCRIPTION
**Pull request by mattmazzola**
Related to server and sdk changes.

There are 4 primary cases, with the inserts occurring in either modals
- Saving directly
- Inserting user input
- Inserting bot action
- Continue

There are still pending bugs related to continuing dialogs and inserting errors being unreliable but no one seems to run into these

### CODE REVIEW

**Comment by mattmazzola** [comment]
I guess the license/cla status is stuck again. Closing and re-opening

**Comment by mattmazzola** [comment]
Something seems to be wrong with Travis queue... re opening again

**Comment by mattmazzola** [comment]
Ah...

Update from Travis
> Slow booting Linux buildsAn update has been postedWe are still investigating. We will post an update as soon as we have more information. Mar 27, 15:57 UTC

File: src/components/modals/LogConversionConflictModal.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                isOpen={this.props.open}
+                className={OF.FontClassNames.mediumPlus}
+                containerClassName="cl-modal cl-modal--medium"
+                onDismiss={() => this.props.onClose()}
+            >
+                <div
+                    className={`cl-modal_header cl-text--error ${OF.FontClassNames.xxLarge} `}
+                    data-testid="log-conversion-conflicts-modal"
+                >
+                    <OF.Icon iconName="Warning" />&nbsp;{formatMessageId(intl, FM.LOGCONVERSIONCONFLICTMODAL_TITLE)}
```
Comment (LarsLiden) [inline src/components/modals/LogConversionConflictModal.tsx]: Slight preference for using <pre> or <span> with padding over &nbsp, but not worth changing

Comment (LarsLiden) [reply src/components/modals/LogConversionConflictModal.tsx]: >   < pre > or < span >

**Pull request closed without merging**